service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfChangeControllerTest.java

```diff
@@ -67,129 +56,256 @@
+                .legalRepresentation("Yes")
+                .solicitor(RespondentSolicitor.builder()
+                    .firstName(solicitor.getForename())
+                    .lastName(solicitor.getSurname().orElse(null))
+                    .email(solicitor.getEmail())
+                    .organisation(org)
+                    .build())
+                .build());
+        }
     }
 
-    @Test
-    void shouldUpdateRespondentRepresentation() {
-
-        final Element<Respondent> respondent1 = element(Respondent.builder()
-            .party(RespondentParty.builder()
-                .firstName("First")
-                .lastName("Respondent")
-                .build())
+    @Nested
+    class Submitted {
+        final String caseName = "Test";
+        final String solicitorEmail = "solicitor@email.com";
+        final String notificationReference = "localhost/" + CASE_ID;
+        final Respondent otherRespondent = Respondent.builder()
+            .legalRepresentation("Yes")
             .solicitor(RespondentSolicitor.builder()
-                .firstName("Tim")
-                .lastName("Brown")
-                .email("tim.brown@test.com")
-                .organisation(organisation("OLD_ORG"))
-                .build())
-            .build());
-
-        final Element<Respondent> respondent2 = element(Respondent.builder()
-            .party(RespondentParty.builder()
-                .firstName("Second")
-                .lastName("Respondent")
-                .build())
-            .build());
-
-        final Element<Respondent> respondent3 = element(Respondent.builder()
-            .party(RespondentParty.builder()
-                .firstName("Third")
-                .lastName("Respondent")
-                .build())
-            .build());
-
-        final ChangeOrganisationRequest changeRequest = ChangeOrganisationRequest.builder()
-            .organisationToAdd(NEW_ORGANISATION)
-            .caseRoleId(caseRoleDynamicList("[SOLICITORA]"))
+                .firstName("Other")
+                .lastName("Solicitor")
+                .email(solicitorEmail)
+                .organisation(Organisation.builder().organisationID("123").build()).build())
             .build();
 
-        final CaseData caseData = CaseData.builder()
+        final CaseData caseDataBefore = CaseData.builder()
+            .id(CASE_ID)
+            .caseName(caseName)
+            .respondents1(wrapElements(
+                otherRespondent,
+                Respondent.builder()
+                .legalRepresentation("Yes")
+                .solicitor(RespondentSolicitor.builder()
+                    .firstName("Old")
+                    .lastName("Solicitor")
+                    .email(solicitorEmail)
+                    .organisation(Organisation.builder().organisationID("123").build()).build())
+                .build()
+            )).build();
+
+        private final CaseData caseData = CaseData.builder()
```

Comment:
inconsistent modifiers for members, other have final only this has private final

to stress that only respondent has changed i would use
caseData = caseDataBefore.toBuilder()
            .respondents1(..........